src/lightning/data/streaming/downloader.py

```diff
@@ -63,8 +65,15 @@ def download_file(cls, remote_filepath: str, local_filepath: str) -> None:
         )
 
 
-# TODO: Add fsspec support
-_DOWNLOADERS = {"s3://": S3Downloader}
+class LocalDownloader(Downloader):
+    @classmethod
+    def download_file(cls, remote_filepath: str, local_filepath: str) -> None:
+        if not os.path.exists(remote_filepath):
+            raise FileNotFoundError("The provided remote_path doesn't exist.")
+        shutil.copy(remote_filepath, local_filepath)
+
+
+_DOWNLOADERS = OrderedDict(**{"s3://": S3Downloader, "": LocalDownloader})
```

Comment:
```suggestion
_DOWNLOADERS = {"s3://": S3Downloader, "": LocalDownloader}
```
Python dicts are already insertion ordered